tensorflow/java/src/test/java/org/tensorflow/SessionTest.java

```diff
@@ -151,7 +151,7 @@ public void failOnUseAfterClose() {
       s.close();
       try {
         s.runner().run();
-        fail("methods on a close()d session should fail");
+        fail("methods on a close() session should fail");
```

Comment:
The `d` there was intentional, to read as the verb `closed`. However, I see it isn't particularly standard.

I don't feel too strongly about the change, but if you'd like to make one perhaps phrase it something like:

"methods on a session should fail after close() is called"
?

Reply:
Thanks! Updated.